### PR TITLE
draft: importing lodash per method

### PR DESCRIPTION
**Pull request by pnavarrc**
```
File sizes after gzip:

  393.1 KB (+287 B)      build/static/js/main.309fa224.chunk.js
  337.34 KB (-13.29 KB)  build/static/js/6.44db3370.chunk.js
  244.53 KB              build/static/js/25.615e7ad7.chunk.js
  138.39 KB (-1 B)       build/static/js/24.07950eac.chunk.js
  125.61 KB (-2 B)       build/static/js/26.47f5c046.chunk.js
  89.15 KB (+13 B)       build/static/js/2.8bde53f1.chunk.js
  75.92 KB (+75.54 KB)   build/static/js/18.8fe1aa89.chunk.js
  43.83 KB (+258 B)      build/static/js/1.b2c9ad72.chunk.js
  26.34 KB (+25.33 KB)   build/static/js/13.f9c1bf3a.chunk.js
  22.51 KB (-125 B)      build/static/js/9.c43ea3ae.chunk.js
  12.78 KB (+120 B)      build/static/js/7.defda86e.chunk.js
  10.6 KB (+1 B)         build/static/js/21.dd3e8fed.chunk.js
  9.08 KB (-2 B)         build/static/js/3.e6638490.chunk.js
  6 KB (+2 B)            build/static/js/20.ded0e5c9.chunk.js
  5.08 KB (-70.83 KB)    build/static/js/17.afba82f7.chunk.js
  5 KB (+26 B)           build/static/js/0.ad814509.chunk.js
  4.74 KB (-21.6 KB)     build/static/js/12.6c42b0a0.chunk.js
  2.61 KB (-20 B)        build/static/js/8.6f9c3198.chunk.js
  2.25 KB (+79 B)        build/static/js/15.e883cafe.chunk.js
  2.17 KB (-2.9 KB)      build/static/js/16.fd460a75.chunk.js
  2.12 KB (+33 B)        build/static/js/10.33aac492.chunk.js
  2.11 KB (-1 B)         build/static/js/19.529b8a74.chunk.js
  1.45 KB (-1 B)         build/static/js/22.aa833138.chunk.js
  1.39 KB (-1 B)         build/static/js/runtime-main.d14ffad8.js
  1 KB (-1.25 KB)        build/static/js/14.deb9b1b6.chunk.js
  775 B                  build/static/css/main.835dbc44.chunk.css
  583 B (-4.17 KB)       build/static/js/11.d403c294.chunk.js
  291 B (+2 B)           build/static/js/23.81dc6f1d.chunk.js
```

**Develop**
image

**This branch**
image

### CODE REVIEW

File: src/common/regions/region_db.ts [hunk 1]
```diff
@@ -1,4 +1,7 @@
-import { sortBy, takeRight, values, Dictionary } from 'lodash';
+import sortBy from 'lodash/sortBy';
+import takeRight from 'lodash/takeRight';
+import values from 'lodash/values';
+import type { Dictionary } from 'lodash';
```
Comment (mikelehen) [inline src/common/regions/region_db.ts:4]: ooooh, I didn't know there was `import type`...  I wonder if that helps with the next.js thing you noticed?

File: src/screens/internal/CompareSnapshots/utils.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { takeRight as _takeRight } from 'lodash';
+import _takeRight from 'lodash/takeRight';
```
Comment (mikelehen) [inline src/screens/internal/CompareSnapshots/utils.ts:1]: why does this work?  Shouldn't it be `takeRight as _takeRight` or whatever?

**Review by mikelehen — APPROVED** [review]
Confused by one thing, but this LGTM if you want to ship it.

File: src/screens/internal/CompareSnapshots/utils.ts (hunk 2 above)
Comment (pnavarrc) [reply src/screens/internal/CompareSnapshots/utils.ts:1]: It's the default export from `lodash/takeRight`, we can name it whatever we want. I should probably rename it to `takeRight` though for consistency

Comment (mikelehen) [reply src/screens/internal/CompareSnapshots/utils.ts:1]: Oh, weird!  Feel free to leave it.  I thought it was an oversight (that somehow still worked, haha... I probably should have looked closer.)

**Pull request merged**